doc/refman/Extraction.tex

```diff
@@ -21,9 +21,14 @@
 
 Before using any of the commands or options described in this chapter,
 the extraction framework should first be loaded explicitly
-via {\tt Require Extraction}. Note that in earlier versions of Coq, these
-commands and options were directly available without any preliminary
-{\tt Require}.
+via {\tt Require Extraction}, or via the more robust
+{\tt Require Coq.extraction.Extraction}.
```

Comment:
From an aesthetic point of view, I tend to prefer `From Coq Require Extraction`, but that should not prevent the patch to be merged, as long as the documentation check succeeds.